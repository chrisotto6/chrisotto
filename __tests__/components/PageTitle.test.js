import { render } from '@testing-library/react'
import PageTitle from '@/components/PageTitle'

it('renders page title unchanged', () => {
  const { container } = render(<PageTitle>{'Test'}</PageTitle>)
  expect(container).toMatchSnapshot()
})
