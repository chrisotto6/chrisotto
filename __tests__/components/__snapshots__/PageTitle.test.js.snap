// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page title unchanged 1`] = `
<div>
  <h1
    class="text-3xl font-extrabold leading-9 tracking-tight text-gray-900 dark:text-gray-100 sm:text-4xl sm:leading-10 md:text-5xl md:leading-14"
  >
    Test
  </h1>
</div>
`;
